lib/plugins/package/lib/zipService.js

```diff
@@ -2,12 +2,15 @@
 
 /* eslint-disable no-use-before-define */
 /* eslint-disable no-param-reassign */
+/* eslint-disable consistent-return */
```

Comment:
Thanks @HyperBrain 👍 Yes, that's a good suggestion. I'll add the comments to the lines.